ansible/roles/config/tasks/main.yaml

```diff
@@ -15,6 +15,11 @@
     dest: "/etc/crictl.yaml"
 
 - name: restart containerd
+  changed_when: False
```

Comment:
This is changing the state on the host right?
```suggestion
```

Reply:
It does and I now wonder if I went too far with this one...

When running the molecule suite I did hit plenty of idempotency issues and had to fix them to see the tests running through.

```
CRITICAL Idempotence test failed because of the following tasks:
*  => config : restart containerd
WARNING  An error occurred during the test sequence action: 'idempotence'. Cleaning up.
```
I should try but I believe the right™ fix here is to use notification and handler;

```
- name: copy crictl config to remote
  template:
    src: "crictl.yaml"
    dest: "/etc/crictl.yaml"
  notify:
    - reload systemd
```

handlers/main.yaml:
```
- name: reload systemd
  command: systemctl daemon-reload
```